charts/operator-wandb/charts/app/values.yaml

```diff
@@ -9,6 +9,14 @@ image:
   pullPolicy: Always
   # pullSecrets: []
 
+glueSingleton:
+  enabled: false
+
+hpa:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 1
```

Comment:
we don't need hpa, we just need to be able to set replica count. Since this is a hack we should just do replace count and do this after based on the design we go with?

Reply:
Does that mean I should just completely remove `charts/operator-wandb/charts/app/templates/hpa.yaml`? 

Otherwise it will cause issues or it isn't used.

Reply:
I think it will be more work to shift in the future tbh.